github/authorizations.go

```diff
@@ -43,6 +43,10 @@ const (
 	ScopeAdminGPGKey    Scope = "admin:gpg_key"
 )
 
+const (
+	mediaTypeOAuthAppPreview = "application/vnd.github.doctor-strange-preview+json"
```

Comment:
Let's please move this to `github.go` around line 138 and also add the appropriate comment for consistency with the rest of the custom media headers.  Thanks!

Reply:
Got it. Let me know if there's anything else we need to cover.